src/main/kotlin/org/bundleproject/json/github/GithubRelease.kt

```diff
@@ -0,0 +1,23 @@
+package org.bundleproject.json.github
+
+data class GithubRelease(
```

Comment:
How much of this data do we actually use? Could this be reduced to the data being used?